x-pack/legacy/plugins/canvas/public/expression_types/arg_types/series_style/simple_template.tsx

```diff
@@ -49,12 +49,14 @@ export const SimpleTemplate: FunctionComponent<Props> = props => {
       {!color || color.length === 0 ? (
         <Fragment>
           <EuiFlexItem grow={false}>
-            <span>Color&nbsp;</span>
+            <EuiText size="s">Color&nbsp;</EuiText>
```

Comment:
Can you also fix the display for when the color is **not** auto? It looks to be bigger than this implementation.

<img width="331" alt="Screen Shot 2019-09-23 at 15 34 09 PM" src="https://user-images.githubusercontent.com/549577/65457653-066f8a80-de1a-11e9-9511-e0aaf1d9d184.png">